src/api/faq/faq.controller.js

```diff
@@ -31,11 +36,54 @@ async function getFAQ(req, res, next) {
       }));
     res.json(finalResponse);
   } catch (error) {
-    if (error instanceof ReferenceError) {
-      res.status(404);
-    }
+    if (error instanceof ReferenceError) res.status(404);
     next(error);
   }
 }
 
-module.exports = { getFAQ };
+async function addFAQ(req, res, next) {
+  try {
+    const {
+      question,
+      answer,
+    } = req.body;
+    if (!question || !answer) throw new ReferenceError('Make sure your request includes a question and answer.');
+    await new FAQ({
+      question,
+      answer,
+    }).save();
+    res.json({
+      status: 200,
+      message: 'FAQ added successfully to DB.',
```

Comment:
Should we include the ID of the created FAQ in the response?

Reply:
Done! Is that what you had in mind or should it be its own property?

Reply:
That's what I had in mind, perfect the way it is now.